diagnostics/convective_transition_diag/convecTransCriticalCollapse_util.py

```diff
@@ -891,7 +894,7 @@ def convecTransCriticalCollapse_plot(argsv1,argsv2,argsv3,argsv4,argsv5,argsv6):
     footnote_str+='\n$w_c$ estimated by fitting (dashed) the average precip. pickup curves for the 3 most probable temperature bins'
     if (OVERLAY_OBS_ON_TOP_OF_MODEL_FIG and p1_obs.size!=0):
         footnote_str+='\nCorresponding results from '+OBS+' (spatial resolution: '+RES+'$^{\circ}$) plotted in gray'
-    ax1.text(s=footnote_str, x=0, y=-0.02, transform=fig_wc.transFigure, ha='left', va='top', fontsize=12)
+    #ax1.text(s=footnote_str, x=0, y=-0.02, transform=fig_wc.transFigure, ha='left', va='top', fontsize=12)
```

Comment:
Is there a need to keep the old line around?

Reply:
Keep it for now before figuring out how to uncomment the line.